ppr-ui/tests/unit/EditCollateral.spec.ts

```diff
@@ -88,6 +89,30 @@ describe('Collateral add tests', () => {
   })
 })
 
+
+describe('Collateral tests for MH', () => {
+  let wrapper: Wrapper<any>
+
+  beforeEach(async () => {
+    await store.dispatch('setRegistrationType', mockedMarriageMH)
+    wrapper = createComponent(-1, false)
+  })
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders with readonly type and manufactured home input', async () => {
+    expect(wrapper.findComponent(EditCollateral).exists()).toBe(true)
+    expect(wrapper.find('#txt-type').exists()).toBe(true)
+    // show read only text box
+    expect(wrapper.find('#txt-man').exists()).toBe(true)
+
+    // no drop down
+    expect(wrapper.find('#txt-type-drop').exists()).toBe(false)
```

Comment:
Don't worry about this for now, but I think it's nice to define these given ids/classes for selectors at the top of the test suite (not for vuetify classes, but for the ones we define ourselves) so that if we change it we only need to do it in one place and I think it's nicer for someone new to the area of code

Reply:
gotcha, I did add one for the drop down